src/server/handlers.rs

```diff
@@ -0,0 +1,46 @@
+use actix_web::{HttpRequest, HttpResponse, Responder};
+
+pub async fn get_handler(_req: HttpRequest) -> impl Responder {
+    HttpResponse::Ok()
+        .content_type("application/json")
+        .body("{}")
+}
+#[actix_rt::test]
+async fn test_index() {
+    use super::*;
+    use actix_web::test;
+    let mut app = test::init_service(
+        App::new().service(web::resource("/1.0/sync/1.5").route(web::post().to(get_handler))),
+    )
+    .await;
+
+    let req = test::TestRequest::post().uri("/1.0/sync/1.5").to_request();
+    let res = test::call_service(&mut app, req).await;
+
+    assert_eq!(res.status(), 200, "/1.0/sync/1.5 should return 200");
+}
+// Negative Tests to to make sure that things don't slip through that shouldn't in regex.
```

Comment:
Again, I'd really like to see these tests exercise the `ClientState::from_request` method rather than just `regex`.